### ISSUE TITLE
Update 5.30.6 - Not all emoji are bigger in size and becomes too small in group and next to fonts

### ISSUE THREAD

**Issue by desmonk** [signalapp/Signal-Android#11981]
- [x] I have searched open and closed issues for duplicates #11978 
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Send emoji and letters as shown in the image below. You can see the end result being inconsistent.

### Steps to reproduce
- Install version 5.30.6
- Use emoji and a message as depicted.
- You will encounter the result as shown here.

**Actual result:** Update has made emoji appearance inconsistent.
**Expected result:** Should have a proper implementation.

### Screenshots

![IMG_002](https://user-images.githubusercontent.com/55331939/153346802-7d24f82f-9765-4abb-b6e2-e9eba3598a03.jpg)

### Device info

**Device:** Multiple
**Android version:** 4-11
**Signal version:** 5.30.6

### Link to debug log
NA

**Comment by cody-signal** [comment]
![Screenshot_1644505011](https://user-images.githubusercontent.com/66265416/153433764-a7b57307-b67d-4a0f-8177-f9ee766020e8.png)

Emojis scale based on how many, up to 5 total. This brings us inline with the behavior on iOS and Desktop. We used to do 8 but the other clients do 5 so now we do too. So if you only have one emoji it's going to be larger than if you have 2-5. Once you have more than 5 it'll go to the size used when text is also provided.

Your other bug report was fixed via remove the auto-download restrictions. I assume you turned those back on? We cache jumobemoji we download so even if you have the restrictions in place, but you've already downloaded them previous, we'll show them jumbo. If you haven't downloaded them already, and the restrictions are in place, we'll show them as non-jumbo or normal message size. From what I can tell (since you didn't provide a log) is that is what's happening. See the above screenshot for what this looks like under no restrictions.

**Comment by desmonk** [comment]
This is correct! Problem now is when restrictions are in place, user will get uneven distribution of emoji and texts as shown here and maybe an alternate method to get new bigger emoji might help.

As this is not possible for a user to know that such restrictions can create this issue. Also, just a request here to have some fallback with old size getting restored instead of them becoming too small and creating confusion.

Hope something gets implemented as this is not a good execution as users won't alter data restrictions or even know that doing so is the solution!

Thank you but this is not a proper solution here as it works only for those don't have such restrictions and the fallback here makes them too small as they are not reverting back to normal size we had before.

In short, the normalized size of non jumbo emoji is destroyed for forever after this update.
